Desktop/components/JASP/Widgets/MainPage.qml

```diff
@@ -178,15 +178,16 @@ Item
 			WebEngineView
 			{
 				id:						resultsView
+				clip:                   true
```

Comment:
I suppose you have tested this fix on the Windows laptop, but why do you need to add this `clip: true`?

Reply:
Well we don't "need" this clip, but because clipping forces the to-be-clipped texture to be rendered separately from the rest of the qml-scenegraph I figured it couldnt hurt to force that.

If it doesn't do anything it would only slow down the whole mess marginally so I think we can just leave it in.

Reply:
Also, yes I tested it ;)